ui/plugin-lifecycle/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "test",
```

Comment:
Let's improve the name and description of this package.

Reply:
Name is changed in initial commit.